### DISCUSSION TITLE
Compliance Operator and FCOS

### DISCUSSION THREAD

**Discussion by forzamehlano** [okd-project/okd#860] [Q&A]
The documentation at https://docs.okd.io/latest/security/compliance_operator/compliance-operator-understanding.html talks about deploying the compliance operator on OKD. I know this isn't specific to OKD but I'm hoping the people in this discussion might have an insight into this...

Deploying the compliance operator works and the compliance scans for OCP4 profiles work great. However, the compliance scans for the rhcos4 profiles fail due to not running rhcos (kinda as expected) and all return NOT_APPLICABLE.

Do equivalent profiles exist for FCOS? I can't seem to find them anywhere. Should I be using something more generic to Fedora itself, or am I missing something more fundamental?

Thanks

**Comment by vrutkovs** [comment]
Sounds like a compliance operator bug
